tv.kodi.Kodi.json

```diff
@@ -1,62 +1,63 @@
 {
     "app-id": "tv.kodi.Kodi",
     "runtime": "org.freedesktop.Platform",
-    "runtime-version": "1.6",
+    "runtime-version": "18.08",
     "sdk": "org.freedesktop.Sdk",
     "sdk-extensions": [
-        "org.freedesktop.Sdk.Extension.openjdk9"
+        "org.freedesktop.Sdk.Extension.openjdk10"
     ],
     "finish-args": [
         "--device=dri",
-        "--filesystem=home",
+        "--filesystem=xdg-music",
+        "--filesystem=xdg-pictures",
+        "--filesystem=xdg-videos",
         "--share=ipc",
         "--share=network",
         "--socket=pulseaudio",
-        "--socket=wayland",
         "--socket=x11",
         "--system-talk-name=org.bluez",
         "--system-talk-name=org.freedesktop.Avahi"
     ],
     "command": "kodi",
-    "post-install": [
-        "sed -i 's|^KODI_DATA.*|KODI_DATA=${HOME}/.var/app/tv.kodi.Kodi/data|' /app/bin/kodi"
-    ],
+    "rename-desktop-file": "kodi.desktop",
+    "rename-icon": "kodi",
     "modules": [
+        "shared-modules/udev/udev-175.json",
+        "shared-modules/python2.7/python-2.7.15.json",
         "deps/avahi.json",
+        "deps/libical.json",
         "deps/bluez.json",
+        "deps/fmt.json",
         "deps/fribidi.json",
         "deps/fstrcmp.json",
         "deps/ghostscript.json",
-        "deps/glu.json",
+        "shared-modules/glu/glu-9.0.0.json",
         "deps/libass.json",
         "deps/libcdio.json",
         "deps/lzo2.json",
         "deps/samba.json",
         "deps/swig.json",
         "deps/taglib.json",
         "deps/tinyxml.json",
-        "deps/udev.json",
         {
             "name": "kodi",
-            "buildsystem": "cmake",
+            "buildsystem": "cmake-ninja",
             "config-opts": [
                 "-DVERBOSE=1",
                 "-DENABLE_INTERNAL_FFMPEG=ON",
                 "-DENABLE_INTERNAL_FLATBUFFERS=ON",
-                "-DENABLE_INTERNAL_FMT=ON",
                 "-DENABLE_INTERNAL_RapidJSON=ON",
                 "-DFLATBUFFERS_URL=build/download/flatbuffers-1.9.0.tar.gz",
                 "-DCROSSGUID_URL=build/download/crossguid-8f399e8bd4.tar.gz",
-                "-DFMT_URL=build/download/fmt-5.1.0.tar.gz",
                 "-DRapidJSON_URL=build/download/rapidjson-1.1.0.tar.gz",
                 "-DFFMPEG_URL=build/download/4.0.2-Leia-Alpha3.tar.gz",
                 "-DLIBDVDCSS_URL=build/download/1.4.1-Leia-Beta-3.tar.gz",
                 "-DLIBDVDREAD_URL=build/download/6.0.0-Leia-Alpha-3.tar.gz",
                 "-DLIBDVDNAV_URL=build/download/libdvdnav-6.0.0-Leia-Alpha-3.tar.gz",
-                "-DJava_JAVA_EXECUTABLE=/usr/lib/sdk/openjdk9/bin/java"
+                "-DJava_JAVA_EXECUTABLE=/usr/lib/sdk/openjdk10/bin/java"
             ],
             "post-install": [
-                "mv /app/share/applications/kodi.desktop /app/share/applications/tv.kodi.Kodi.desktop"
+                "sed -i 's|^KODI_DATA.*|KODI_DATA=${HOME}/.var/app/tv.kodi.Kodi/data|' /app/bin/kodi"
```

Comment:
`${XDG_DATA_HOME}`

Reply:
ah yes, will do

Reply:
this is already done in the kodi-sh-in.patch above

Reply:
oh, yes, of course